deploy-board/deploy_board/templates/configs/new_capacity.html

```diff
@@ -188,6 +188,7 @@ <h4 class="panel-title pull-left pointer-cursor">Basic Settings</h4>
             clusterInfo['capacity'] = Number(this.instanceCount);
             clusterInfo['archName'] = capacityCreationInfo['defaultArch'];
             clusterInfo['baseImageId'] = this.baseImageValue;
+            clusterInfo['useLaunchTemplate'] = false;
```

Comment:
let me know if directly putting false here is appropriate, or I should do this in cluster_view.py. Thanks

Reply:
It's better to put it in the clusters_view.py and pass the variable through. I think it will have a permanent value of "true" once the migration is complete, so it is best to put it in the right place for now.

Reply:
Also, since it is separate in the advanced review, that would reduce it to just 1 location where there is the hardcoded assignment, which will reduce the chance of errors in the future.